### PR TITLE
adding .adoc files for documentation

### PR DESCRIPTION
**Pull request by vinagarw272001**
# Changes

# Submitter Checklist

- [ ] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [ ] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)  
- [ ] Release notes block has been filled in, or marked NONE

See [the contributor guide](https://github.com/shipwright-io/build/blob/main/CONTRIBUTING.md)
for details on coding conventions, github and prow interactions, and the code review process.

# Release Notes

### CODE REVIEW

File: docs-adoc/local_source_upload.adoc
Comment (otaviof) [inline docs-adoc/local_source_upload.adoc:1]: @vinagarw272001 could you please use `git mv` to mark those files as renamed?

**Review by otaviof — CHANGES_REQUESTED** [review]
Nice one, @vinagarw272001!

**Comment by otaviof** [comment]
The reason for this change is shared [here](https://github.com/shipwright-io/build/pull/1324#issuecomment-1607410801), please consider.

**Comment by coreydaley** [comment]
/close
discussed by community and decided against it

**Pull request closed without merging**